docs/reference/query-rules/apis/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[query-rules-apis]]
+== Query rules APIs
+
+preview::[]
+
+++++
+<titleabbrev>Query rules APIs</titleabbrev>
+++++
+
+---
+
+Use query rules APIs to manage tasks and resources related to query rules.
```

Comment:
That's fair, thank you! I wonder if, from a PR perspective, additional details should come later when [Rule Query](https://github.com/elastic/elasticsearch/pull/97250) is merged (it will make the information + linking more useful).

Reply:
Sounds good 👍

Reply:
I'd say to complete this so we don't have to revisit it, but happy to defer to later PRs. Will there need to be linking from these docs to some info included in the Rule Query PR?

Reply:
> Will there need to be linking from these docs to some info included in the Rule Query PR?

I suspect we'll want to, because the rule query is the only reason we'll want to use these APIs.

Reply:
@carlosdelest Since the followup docs PR will be imminent I'm going to go ahead and get these docs in, and create a new one for rule query. Thanks!